syntax = "proto3";
package cdm;
option java_package = "com.kitware.physiology.cdm";
option optimize_for = SPEED;

import "cdm/Engine.proto";

message ScenarioData
{
  string                           Name                      = 1;
  string                           Description               = 2;
  EngineInitializationData         StartType                 = 3;
  DataRequestManagerData           DataRequestManager        = 4;
  repeated AnyActionData           AnyAction                 = 5;
}
